tests/simulation/test_simulator.py

```diff
@@ -61,6 +62,7 @@ def test_two_nodes(self):
         self.assertTrue(conn12.is_connected)
         self.assertTipsEqual(manager1, manager2)
 
+    @pytest.mark.flaky(max_runs=5, min_passes=1)
```

Comment:
I feel that sync-v2 would not affect how flaky this test is, so we could remove this marker without prejudice.

Reply:
I removed this from the base test and moved it to the sync-v2 base so it doesn't affect how the test runs on sync-v1, but I still left the flaky mark on the sync-v2 tests.